src/pages/[id]/index.js

```diff
@@ -50,11 +47,25 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
     fillActiveTasksArray();
   }, [fillActiveTasksArray]);
 
+  if (errorMessage) {
+    return <NotFound errorMsg={errorMessage} />;
+  }
+
+  if (!isSuperUser) {
+    if (!apiCalledToVerifySuperUser) {
```

Comment:
Nit: Could have been just one If here.

Reply:
merged both the conditions into one. Thanks.